src/renderer/src/groups/foyer.ts

```diff
@@ -133,19 +141,23 @@ export class FoyerStore {
 
     stream.addMessage(message);
     if (message.payload.type == 'Msg') {
-      //   if (isWeaveContext()) {
-      //     this.weaveClient.notifyFrame([
-      //       {
-      //         title: `message from ${encodeHashToBase64(message.from)}`,
-      //         body: message.payload.text,
-      //         notification_type: 'message',
-      //         icon_src: undefined,
-      //         urgency: 'high',
-      //         timestamp: message.payload.created,
-      //       },
-      //     ]);
-      //   }
-      if (encodeHashToBase64(message.from) != this.myPubKeyB64) {
+      const mainWindowFocused = await window.electronAPI.isMainWindowFocused();
+      let b64From = encodeHashToBase64(message.from);
+      if (!mainWindowFocused) {
+        const profile = await toPromise(this.profilesStore.profiles.get(message.from));
```

Comment:
This notification logic should probably also only happen if it's not a message from oneself, right? So only if `b64From != this.myPubKeyB64`